lib/ma_crossover/events/life_start.js

```diff
@@ -1,6 +1,12 @@
 'use strict'
 
+const fs = require('fs')
+const os = require('os')
 const HFI = require('bfx-hf-indicators')
+const checkHeader = require('../util/check_header')
+const dir = `${os.homedir()}/.honeyframework/ma_crossover.csv`
+
+const HEADERS_FOR_CSV = ['GID', 'TYPE', 'DATETIME', 'PRICE', 'PRICE_TYPE', 'MA_TYPE', 'MA_VALUE']
```

Comment:
how about a more general approach, where you can start the HF with a flag like `ALGO_LOG=true` and it would create CSVs for all algo orders?

could be encapsulated in a general function and available by a event like `await emit('exec:log_csv', gid, [value1, value2 ...])`

and in cases where logging is disabled, the general function immediately returns...

Reply:
we could add the base for it in this PR with the logging for ma cross, and then add other algos as well in seperate follow up PRs